tide/core/Content.cpp

```diff
@@ -138,6 +138,12 @@ Content::Interaction Content::getInteractionPolicy() const
     return Content::Interaction::AUTO;
 }
 
+const QString& Content::getQmlControls() const
+{
+    static const QString empty;
+    return empty;
+}
```

Comment:
what is the reason for this function ? Why not returning a QString, you need a ref ?